content/docs/installation.md

```diff
@@ -4,15 +4,140 @@ description: Get up and running on Linux and a variety of container platforms
+
+  1. Create the `k8s-with-rbac/falco-config` directory.
+  2. Copy the required configuration from this GitHub repository to the `k8s-with-rbac/falco-config/` directory. 
+
+Do not modify the original files. Use the files you copied to make any configuration changes. 
+
+```shell
+mkdir -p k8s-with-rbac/falco-config
+k8s-using-daemonset$ cp ../../falco.yaml k8s-with-rbac/falco-config/
+k8s-using-daemonset$ cp ../../rules/falco_rules.* k8s-with-rbac/falco-config/
+k8s-using-daemonset$ cp ../../rules/k8s_audit_rules.yaml k8s-with-rbac/falco-config/
+```
+
+5. Add the custom rules for your environment to the `falco_rules.local.yaml` file and they will be picked up by Falco at start time. You can also modify the `falco.yaml` file to change any [configuration options](configuration/) required for your deployment. Create the configMap as follows:
+```shell
+kubectl create configmap falco-config --from-file=k8s-with-rbac/falco-config
+```
+
+6. With the dependencies of the configMap created, you can now create the DaemonSet.
+```shell
+kubectl apply -f k8s-with-rbac/falco-daemonset-configmap.yaml
+```
+
+7. Verify Falco started correctly. To do so, check the status of the Falco pods in the corresponding log files.
+```shell
+kubectl logs -l app=falco
+```
+
+### Minkube
+
+The easiest way to use Falco on Kubernetes in a local environment is on [Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/). Both the Kubernetes YAML manifests and the Helm chart are regularly tested with Minikube.
+
+#### Minikube Kernel Module
+
+When running `minikube` with the default `--driver` arguments, Minikube creates a VM that runs the various Kubernetes services and a container framework to run Pods, etc. Generally, it's not possible to build the Falco kernel module directly on the Minikube VM, as the VM doesn't include the kernel headers for the running kernel.
+
+To address this, starting with falco 0.13.1 we pre-build kernel modules for the last 10 Minikube versions and make them available at https://s3.amazonaws.com/download.draios.com. This allows the download fallback step to succeed with a loadable kernel module.
+
+Going forward, we'll continue to support 10 most recent versions of Minikube with each new Falco release. We currently retain previously-built kernel modules for download, so we will continue to provide limited historical support as well.
+
+### GKE
+
+Google Kubernetes Engine's (GKE) default operating system for worker node pools is Container-Optimized OS (COS), a security enhanced operating system that limits access to certain parts of the underlying OS. Because of these security enhancements, Falco cannot insert its kernel module in order to process events for system calls. COS does provide the ability to leverage eBPF (extended Berkeley Packet Filter) to provide the stream of system calls to the Falco engine. eBPF is currently only supported on GKE and COS.
```

Comment:
```suggestion
Google Kubernetes Engine (GKE) uses Container-Optimized OS (COS) as the default operating system for its worker node pools. COS is a security-enhanced operating system that limits access to certain parts of the underlying OS. Because of this security constraint, Falco cannot insert its kernel module to process events for system calls. However, COS provides the ability to leverage eBPF (extended Berkeley Packet Filter) to supply the stream of system calls to the Falco engine. eBPF is currently supported only on GKE and COS.
```